model/atmosphere/dycore/tests/dycore/stencil_tests/test_vertically_implicit_dycore_solver_at_corrector_step.py

```diff
@@ -370,8 +399,15 @@ def reference(
             exner_dynamical_increment=exner_dynamical_increment,
         )
 
-    @pytest.fixture
-    def input_data(self, grid: base.Grid) -> dict[str, gtx.Field | state_utils.ScalarType]:
+    @pytest.fixture(
+        params=[
+            {"at_first_substep": afs, "at_last_substep": als, "lprep_adv": la}
```

Comment:
To maybe save some unnecessary runs, `at_first_substep` and `at_last_substep` cannot be `true` at the same time